Stop tracking managedField on create/update

liggitt opened the pull request:
**What type of PR is this?**
/kind failing-test

**What this PR does / why we need it**:
Stops tracking managedField on create/update to clean up CI signal (current pass rate is ~30%).

**Which issue(s) this PR fixes**:
Fixes #87131

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

/cc @jennybuckley 
/priority critical-urgent
/sig api-machinery

droslean commented:
/lgtm

jennybuckley commented:
We can also address this by changing the test temporarily, like in https://github.com/kubernetes/kubernetes/pull/87204, which might be preferable since kubernetes-sigs/structured-merge-diff#145 might take a little while to get into k8s/k8s

liggitt commented:
until https://github.com/kubernetes-sigs/structured-merge-diff/pull/145 makes it in, I don't think we should be populating managedFields by default

liggitt committed:
Set managedField probability to 0%

Lowers probability of managedField population on create/update to 0%
until serialization/normalization issues are resolved

liggitt commented:
This can be re-raised once https://github.com/kubernetes-sigs/structured-merge-diff/pull/145 is resolved and updated in kube

liggitt commented:
switched from a revert to a simple lowering of the probability to 0%

droslean commented:
/lgtm

lavalamp commented:
/lgtm

jennybuckley commented:
/retest

liggitt commented:
/retest
